docs/cookbook/meetingPlanner.js

```diff
@@ -1,46 +1,41 @@
 // Display local time zone and three others
-const here = Temporal.now.timeZone();
-const now = Temporal.now.absolute();
+const now = Temporal.now.localDateTime();
 const timeZones = [
-  { name: 'Here', tz: here },
+  { name: 'Here', tz: now.timeZone() },
   { name: 'New York', tz: Temporal.TimeZone.from('America/New_York') },
   { name: 'London', tz: Temporal.TimeZone.from('Europe/London') },
   { name: 'Tokyo', tz: Temporal.TimeZone.from('Asia/Tokyo') }
 ];
 
 // Start the table at midnight local time
-const calendarNow = now.inTimeZone(here);
-const startTime = calendarNow
-  .getDate()
-  .withTime(Temporal.Time.from('00:00')) // midnight
-  .inTimeZone(here);
+const startTime = now.with({ ...Temporal.Time.from('00:00').getFields() }); // midnight
```

Comment:
Yep!  This was my mistake that prompted me to file #713. Already fixed it on my end, but was waiting to push until your debug changes landed so that CI will pass when I push. ;-)  I'll get this pushed today or tomorrow.

Reply:
fixed now